src/ts/components/Footer.ts

```diff
@@ -0,0 +1,16 @@
+import { createElem } from '../utilities';
+
+export default class Footer {
+  drawFooter(parent: HTMLElement): void {
```

Comment:
can be refactored as string template
also can be put in markup directly without js